core/src/main/java/com/novoda/noplayer/internal/mediaplayer/forwarder/BufferHeartbeatListener.java

```diff
@@ -0,0 +1,23 @@
+package com.novoda.noplayer.internal.mediaplayer.forwarder;
+
+import com.novoda.noplayer.Player;
+import com.novoda.noplayer.internal.mediaplayer.CheckBufferHeartbeatCallback;
+
+class BufferHeartbeatListener implements CheckBufferHeartbeatCallback.BufferListener {
```

Comment:
Yes, this was in the `mediaplayer.forwarder` package before. Now it's been moved to `internal.mediaplayer.forwarder`.